tests/unit/backend/corpora/api_server/curator/collection/test_curation_collections.py

```diff
@@ -185,6 +185,134 @@ def test__get_only_private_collections_with_auth__OK(self):
+                    "tombstone": False,
+                    "x_approximate_distribution": "NORMAL",
+                    "x_normalization": "test_x_normalization",
+                }
+            ],
+            "description": "test_description",
+            "id": "test_collection_id",
+            "links": [
+                {"link_name": "test_doi_link_name", "link_type": "DOI", "link_url": "http://test_doi_url.place"},
+                {"link_name": None, "link_type": "DOI", "link_url": "http://test_no_link_name_doi_url.place"},
+                {
+                    "link_name": "test_raw_data_link_name",
+                    "link_type": "RAW_DATA",
+                    "link_url": "http://test_raw_data_url.place",
+                },
+                {"link_name": None, "link_type": "RAW_DATA", "link_url": "http://test_no_link_name_raw_data_url.place"},
+                {
+                    "link_name": "test_protocol_link_name",
+                    "link_type": "PROTOCOL",
+                    "link_url": "http://test_protocol_url.place",
+                },
+                {"link_name": None, "link_type": "PROTOCOL", "link_url": "http://test_no_link_name_protocol_url.place"},
+                {
+                    "link_name": "test_lab_website_link_name",
+                    "link_type": "LAB_WEBSITE",
+                    "link_url": "http://test_lab_website_url.place",
+                },
+                {
+                    "link_name": None,
+                    "link_type": "LAB_WEBSITE",
+                    "link_url": "http://test_no_link_name_lab_website_url.place",
+                },
+                {"link_name": "test_other_link_name", "link_type": "OTHER", "link_url": "http://test_other_url.place"},
+                {"link_name": None, "link_type": "OTHER", "link_url": "http://test_no_link_name_other_url.place"},
+                {
+                    "link_name": "test_data_source_link_name",
+                    "link_type": "DATA_SOURCE",
+                    "link_url": "http://test_data_source_url.place",
+                },
+                {
+                    "link_name": None,
+                    "link_type": "DATA_SOURCE",
+                    "link_url": "http://test_no_link_name_data_source_url.place",
+                },
+            ],
+            "name": "test_collection_name",
+            "published_at": None,
+            "publisher_metadata": None,
+            "revised_at": None,
+            "revision_of": None,
+            "tombstone": False,
+            "visibility": "PUBLIC",
+        }
+
+        res_body = res.json
+        del res_body["created_at"]  # too finicky; ignore
+        self.assertDictEqual(expected_body, res_body)
+
+    def test__get_private_collection__OK(self):
+        res = self.app.get("/curation/v1/collections/test_collection_id_revision")
+        self.assertEqual(200, res.status_code)
```

Comment:
check access_type

Reply:
I do that in the last two tests. Here I'm just focused on private v public.

Reply:
fixed to check access type on private collection call with auth provided ✅